# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#feedback
float32[] position
float32[] rate
float32 time
float32 gripper_feed_value
float32 gripper_feed_rate

